src/Hedgehog/LinqSupport.fs

```diff
@@ -0,0 +1,70 @@
+﻿namespace Hedgehog
+
+open System
+open System.Runtime.CompilerServices
+
+open Hedgehog
+
+[<Extension>]
+module GenLinqSupport =
+
+    // Support for `where`:
+    [<Extension>]
+    [<CompiledName("Where")>]
+    let where (g : Gen<'a>) (f : Func<'a, bool>) : Gen<'a> =
+        Gen.filter (fun x -> f.Invoke x) g
+
+    [<Extension>]
+    [<CompiledName("Select")>]
+    let select (g : Gen<'a>) (f : Func<'a, 'b>) : Gen<'b> =
+        Gen.map (fun x -> f.Invoke x) g
+
+    [<Extension>]
+    [<CompiledName("SelectMany")>]
+    let bind2 (g : Gen<'a>) (f : Func<'a, Gen<'b>>) (proj : Func<'a, 'b, 'c>) : Gen<'c> =
+        gen {
+            let! a = g
+            let! b = f.Invoke a
+            return proj.Invoke(a, b)
+        }
+
+[<Extension>]
+module PropertyLinqSupport =
+
+    [<Extension>]
+    [<CompiledName("Where")>]
+    let where (p : Property<'a>) (f : Func<'a, bool>) : Property<'a> =
+        Property.filter (fun x -> f.Invoke x) p
+
+    [<Extension>]
+    [<CompiledName("Select")>]
+    let select (p : Property<'a>) (f : Func<'a, 'b>) : Property<'b> =
+        Property.map (fun x -> f.Invoke x) p
+
+    [<Extension>]
+    [<CompiledName("SelectMany")>]
+    let bind2 (pa : Property<'a>) (f : Func<'a, Property<'b>>) (proj : Func<'a, 'b, 'c>) : Property<'c> =
+        Property.bind pa (fun a -> Property.bind (f.Invoke a) (fun b -> Property.success (proj.Invoke(a, b))))
```

Comment:
Nitpick: Throughout the F# codebase we add a space between the member/function and its argument. :neckbeard: (near `proj.Invoke`)